gate-web/src/test/java/com/netflix/spinnaker/gate/controllers/InvokePipelineConfigTest.java

```diff
@@ -517,7 +519,8 @@ private void simulateFront50Response(ResponseDefinitionBuilder responseDefinitio
     wmFront50.stubFor(
         WireMock.get(urlPathEqualTo("/pipelines/" + APPLICATION + "/name/" + PIPELINE_NAME))
             .withQueryParam("refresh", anythingPattern)
-            .willReturn(responseDefinitionBuilder));
+            //            .willReturn(responseDefinitionBuilder));
```

Comment:
Please remove commented out code.

Is this change something we could do in a separate commit before the retrofit2 changes (like removing the Front50Service bean)?  Why doesn't `.willReturn(responseDefinitionBuilder)` work?

Reply:
It was supposed to be reverted after fixing an issue related to Call being null. I will revert.